src/More/zendeskAPI.ts

```diff
@@ -0,0 +1,75 @@
+import env from "react-native-config"
+
+const defaultHeaders = {
+  "content-type": "application/json",
+}
+
+type EnvironmentProps = {
```

Comment:
what do you think about using an interface here?

Reply:
Not sure why? I'm not opposed but since the most notable difference from an interface and a type(that I know of) is that re-declarations result in merging I try to use types for final declarations. But definitely up for it.

Reply:
cool, thanks for the reasoning. I also dont have a strong preference.